### ISSUE TITLE
S-500 SAM/ABM Addition (Accurate Specs Not Known/Speculative)

### ISSUE THREAD

**Issue by HyperGopher** [PygmalionOfCyprus/cmo-db-requests#315]
### DB Selector

Both

### Hypothetical

No

### Name

55R6M Triumfator-M

### Facility Type

SAM

### Country

Russia

### Service

Air Defense

### In Commission

2022

### Crew

8

### Dispersion Radius (m)

20

### Sensors

1x **_91N6A(M)_** AESA upgrade of the previous Big Bird D - acquisition and battle management radar, a modification of the 91N6 (Big Bird) towed by the BAZ-6403.01 8x8 tractor
1x **_96L6-TsP_** acquisition radar, an AESA upgraded version of the previous 96L6 (Cheese Board) on BAZ-69096 10x10
1x **_76T6_** AESA multimode anti-air engagement radar on BAZ-6909-022 8x8 (new design)
1x **_77T6_** AESA ABM engagement radar on BAZ-69096 10x10 (new design)
1x **_77P6_** launch vehicle, based on the BAZ-69096 10x10 truck (for 77N6/77N6-1 missiles)
1x 5P85SE2 launch vehicle (for 40N6M missiles)
1x **_55K6MA_** command post, based on BAZ-69092-12 6x6 (for air defense)
1x **_85Zh6-2_** command posts, based on BAZ-69092-12 6x6 (for ABM defense)

### Mounts

4x 77P6 Tel (2x 77N6-N Missile per Tel)
4x 77P6 Tel (2x 77N6-N1 Missile per Tel)
4x 5P85SE2 Tel (4x 40N6(M) Missile per Tel)
12 Tel Total

### Magazines

8x 77N6 Missiles
8x 77N6-1 Missiles
16x 40N6M Missiles
(32 missiles total)

### Comments

First battery reported operational in early 2022. Max missile range against ballistic missile is reported at ~330nm and or at an altitude of ~110nm, and against aircraft the max range is ~280nm at an altitude of ~110nm. All info based off reports/websites online that are available when searching for S-500 info.

**Radar Ranges for _Ballistic Missile_ targets (estimated from reports):** (Source:  https://www.armyrecognition.com/russia_russian_missile_system_vehicle_uk/s-500_prometheus_55r6m_triumfator-m_air_defense_missile_data_pictures_video.html)

1. 91N6A(M) fully AESA S-band upgrade to Big Bird D (estimated range 1500-2000nm) Early warning/Acquisition Radar
2. 96L6-TsP (fully AESA C-band upgrade to Chess Board radar) estimated range of 600-1000nm (Acquisition radar)
3. 76T6 new AESA (X-band) multimode engagement radar (estimated range of 400nm)
4. 77T6 Yanisei AESA new (X-band?) ABM engagement radar (500nm)

**Radar Ranges for aircraft type targets (estimated from reports):** Source: https://www.19fortyfive.com/2021/07/natos-worst-nightmare-russias-s-500-air-defense-system-has-arrived/) 

1. **_91N6A(M)_** - 600nm
2. **_96L6-TsP_** - 350nm
3. _**76T6**_ - 300nm
4. **_77T6 Yanisei_** - 400nm

**When all radars are synchronized in a battery, "the max radar range reaches a radius of 3,000 km (1,300 km for the EPR 0.1 square meter"  Or 3,000km for a 1m2 target. So 702nm for a 0.1m2 target and 1620nm for a 1m2 target (https://en.wikipedia.org/wiki/S-500_missile_system)

The S-500 is designed for intercepting and destroying [intercontinental ballistic missiles], as well as [hypersonic cruise missiles and aircraft, for [air defense] against [Airborne Early Warning and Control]and for jamming aircraft. With a planned range of 600 km (370 mi) for anti-ballistic missile (ABM) and 500 km (310 mi) for air defense the S-500 would be able to detect and simultaneously engage up to 10 ballistic hypersonic targets flying at a speed of 5 kilometers per second (3.1 mi/s; 18,000 km/h; 11,000 mph) to a limit of 7 km/s (4.3 mi/s; 25,000 km/h; 16,000 mph). It also aims at destroying hypersonic cruise missiles and other aerial targets at speeds of higher than Mach 5, as well as spacecraft.[[citation needed. The altitude of a target engaged can be [as high as 180–200 km (110–120 mi)." (https://en.wikipedia.org/wiki/S-500_missile_system)

**New Missiles Used:**

1. **77N6-N1** - Max range 330nm for ballistic target, 280nm for aircraft. Hit-To-Kill (HTK detonation). Alleged active radar seeker + IR seeker. Guided by 77T6 ABM when engaging ballistic missiles. Guided by 76T6 when engaging aircraft. Uses 91N6M for early warning and 96L6-TsP for acquisition. Max target speed (Mach 21)
2. **_77N6-N_** - Max range of 280nm for ballistic target, 220nm for aircraft. Uses HTK detonation. Alleged active radar + IR seeker. Guided by 77T6 ABM for ballistic targets and 76T6 for aircraft. Uses 91N6M for early warning and 96L6-TsP for acquisition. Max Target Speed (Mach 16)
3. **_40N6(M)_** - improved version of 40N6 with (supposed) AESA active radar seeker and engine upgrade. 280nm range for aircraft. Guided by 76T6 multimode engagement radar, 91N6M early warning, and 96L6-TsP acquisition radars) FRAGMENTATION detonation fuse. Max Target Speed (Mach 15)

*Can engage up to 10 hypersonic ballistic missiles simultaneously

"The S-500 would be able to detect and simultaneously engage up to 10 ballistic hypersonic targets flying at a speed of 5 kilometers per second (3.1 mi/s; 18,000 km/h; 11,000 mph ~mach 15) to a limit of 7 km/s (4.3 mi/s; 25,000 km/h; 16,000 mph) ~mach 21 (https://en.wikipedia.org/wiki/S-500_missile_system)

*S-500 can be guided during mid-course flight by the Voronezh-DM/Voronezh-VP, and Don-2 radars if its semi-active receiver becomes blind. "reports by Almaz-Antey say that the external target-designation system will be capable of mid-early flight portion interceptions of enemy ballistic missiles" (https://en.wikipedia.org/wiki/S-500_missile_system#cite_note-rg.ru-22)

### Sources

https://missilethreat.csis.org/defsys/s-500-prometheus/
https://www.armyrecognition.com/russia_russian_missile_system_vehicle_uk/s-500_prometheus_55r6m_triumfator-m_air_defense_missile_data_pictures_video.html
https://www.deagel.com/Artillery%20Systems/S-500/a002423
https://www.globalsecurity.org/wmd/world/russia/77t6.htm
https://cybershafarat.com/2021/10/15/russian-army-s-500-defense-system/
https://www.19fortyfive.com/2021/07/natos-worst-nightmare-russias-s-500-air-defense-system-has-arrived/
https://en.wikipedia.org/wiki/S-500_missile_system#cite_note-rg.ru-22

**Comment by HyperGopher** [comment]
Sorry, meant to label CWDB only. Edited it so should be fixed now.

**Comment by PygmalionOfCyprus** [comment]
Technically a duplicate of #208 but has more info so will leave this guy open and tag/close the other.

**Comment by PygmalionOfCyprus** [comment]
Oi. Github. Duplicate assignment, please. I command you.

Duplicate of #208

**Comment by HyperGopher** [comment]
Unfortunately I feel as if this request is way too hypothetical in nature,
even though the weapon itself is currently in service. It's just that the
ranges let alone more detailed specs of all the radars and missiles are not
known and wont be for at least multiple years if not 5-10+ years. We can
only speculate. There are too many different radars involved and possibly
more than 3 missiles. This is a very hard system to speculate or even work
on.

Basically all we know is max ABM range of 600km and max Anti-air range of
500km.
Max ABM synchronized radars have range of 3000km(1.5m), or 1300km (0.1m2)
in anti-air mode.
Everything is not complete let alone verified.

So again, I fully understand if you guys wont be able to manage integrating
this into the game in the foreseeable future.
